Explorer/Assets/DCL/PerformanceAndDiagnostics/Diagnostics/ReportsHandling/ReportCategory.cs

```diff
@@ -149,6 +149,11 @@ public static class ReportCategory
 
         public const string AUTHENTICATION = nameof(AUTHENTICATION);
 
+        /// <summary>
+        ///     Generic SDK components category
+        /// </summary>
+        public const string SDK_COMPONENT = nameof(SDK_COMPONENT);
```

Comment:
why not `AVATAR_ATTACH`?

Reply:
totally, it's better, already changed it!